### DISCUSSION TITLE
Feature Request: Set per-organization or per-repo email address for PRs (web actions)

### DISCUSSION THREAD

**Discussion by tlsimmons88** [community/community#17551] [Pull Requests]
Note:  This is copied from: https://github.community/t/feature-request-set-per-organization-or-per-repo-email-address-for-prs-web-actions/1759

Per @liligalante request I have moved it here.

I use a single GitHub account for both work and personal projects. I would like to be able to configure GitHub to use my work email address for PRs to repositories in my employer’s organization, and my GitHub privacy-protected email for all other PRs.

This could be accomplished by either a per-organization email address setting, or a per-repository email address setting. I can see benefits and drawbacks of either approach. In my current situation, a per-organization setting and a default for non-organization repos would be the most convenient, but I could imagine someone who works on multiple open-source project.

**Comment by stigmergist** [comment]
Actually, you can now route: https://github.com/settings/notifications
